PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -342,6 +341,10 @@ public function exchange_declare(
         $arguments = null,
         $ticket = null
     ) {
+        if (!$this->is_open) {
+            throw new AMQPRuntimeException('Channel connection is closed.');
```

Comment:
The end result is the same, but the message I currently have, IMO, is more accurate.  It gives the exact reason why the operation could not be performed, which helps with debugging and such.